mirgecom/simutil.py

```diff
@@ -134,8 +134,16 @@ def get_sim_timestep(discr, state, t, dt, cfl, t_final, constant_cfl=False):
 
 
 def write_visfile(discr, io_fields, visualizer, vizname,
-                  step=0, t=0, overwrite=False, vis_timer=None):
-    """Write VTK output for the fields specified in *io_fields*.
+                  step=0, t=0, overwrite=False, vis_timer=None,
+                  mpi_communicator=None):
```

Comment:
Since `comm` is used throughout, I would stick with `comm` instead of `mpi_communicator`